react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      secondElapsed: 0,
+      secondWhenPause: null
+    };
+    this.myInterval = null;
```

Comment:
Typically don't want to use `my` in variable names, something like just `this.interval` or `this.intervalId` would be better.